package.json

```diff
@@ -2,7 +2,7 @@
     "name": "root",
     "private": true,
     "scripts": {
-        "test": "lerna run test",
+        "test": "lerna run test:ci",
```

Comment:
should be `lerna run test`
and add another one called "test:ci": "lerna run test:ci"